ddtrace/contrib/redis/patch.py

```diff
@@ -119,6 +121,20 @@ def unpatch():
                 unwrap(redis.asyncio.cluster.ClusterPipeline, "execute")
 
 
+def _run_redis_command(span, func, args, kwargs):
+    parsed_command = stringify_cache_args(args)
+    redis_command = parsed_command.split(" ")[0]
+    try:
+        result = func(*args, **kwargs)
+        if redis_command in ROW_RETURNING_COMMANDS:
+            span.set_metric(db.ROWCOUNT, determine_row_count(redis_command=redis_command, result=result))
+        return result
+    except Exception:
+        if redis_command in ROW_RETURNING_COMMANDS:
+            span.set_metric(db.ROWCOUNT, 0)
```

Comment:
the async helper does a core.dispatch to set this metric, but we don't here. do we need the dispatch? or should we use it in both places?